system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -80,6 +168,7 @@ const TopSites = props => (<section>
   </ul>
 </section>);
 
-module.exports = connect(state => ({TopSites: state.TopSites}))(TopSites);
+module.exports = connect(state => ({TopSites: state.TopSites}))(TopSitesPerfTimer);
 module.exports._unconnected = TopSites;
```

Comment:
The connected component is `TopSitesPerfTimer`, so the unconnected one would seem to imply it's the `TopSitesPerfTimer`